web/frontend/src/components/slam-2d-render.vue

```diff
@@ -192,6 +310,36 @@ onUnmounted(() => {
   disposeScene();
 });
 
+watch(() => [props.destVector!.z, props.destVector!.x, props.destExists], async () => {
```

Comment:
needs to be `props.destVector!.z, props.destVector!.x` otherwise the `watch` method is not triggered properly :/